suite-native/device-manager/src/components/DeviceManagerModal.tsx

```diff
@@ -1,47 +1,63 @@
 import { GestureResponderEvent, Modal, Pressable } from 'react-native';
 import { ReactNode } from 'react';
 import { useSafeAreaInsets, EdgeInsets } from 'react-native-safe-area-context';
-import Animated, { SlideInUp } from 'react-native-reanimated';
+import Animated, { FadeIn, SlideInUp } from 'react-native-reanimated';
+import { useSelector } from 'react-redux';
 
-import { ScreenHeaderWrapper, VStack } from '@suite-native/atoms';
+import { ScreenHeaderWrapper, Box, HStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { selectDevice } from '@suite-common/wallet-core';
+import { AcquiredDevice } from '@suite-common/suite-types';
 
-import { DeviceSwitch } from './DeviceSwitch';
 import { useDeviceManager } from '../hooks/useDeviceManager';
+import { DeviceItemContent } from './DeviceItemContent';
 
 type DeviceManagerModalProps = {
     children: ReactNode;
+    customSwitchRighView?: ReactNode;
+    onClose?: () => void;
 };
 
-const MANAGER_MODAL_BOTTOM_RADIUS = 20;
+export const MANAGER_MODAL_BOTTOM_RADIUS = 12;
 
 const modalBackgroundOverlayStyle = prepareNativeStyle(utils => ({
     flex: 1,
     backgroundColor: utils.transparentize(0.3, utils.colors.backgroundNeutralBold),
 }));
 
-const deviceManagerModalWrapperStyle = prepareNativeStyle<{ insets: EdgeInsets }>(
+const deviceManagerModalWrapperStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation0,
+    borderBottomLeftRadius: MANAGER_MODAL_BOTTOM_RADIUS,
+    borderBottomRightRadius: MANAGER_MODAL_BOTTOM_RADIUS,
+}));
+
+const deviceSwitchWrapperStyle = prepareNativeStyle<{ insets: EdgeInsets }>(
     (utils, { insets }) => ({
-        paddingTop: Math.max(insets.top, utils.spacings.small),
+        paddingTop: insets.top + utils.spacings.large,
```

Comment:
Why did you decided to add these two values? The padding might be huge on some specific screens.

Reply:
this is to have consistent background color. 
deviceManagerModalWrapperStyle sets it and deviceSwitchWrapperStyle layouts content and needs to have offset from the top

<img width="501" alt="Screenshot 2024-05-10 at 10 51 58" src="https://github.com/trezor/trezor-suite/assets/2011829/0fa0ad5a-c263-444b-ab1b-45037a81ad67">

Reply:
5825a46c34

Reply:
this PR does not fix it right? There is no change :D.

I was afraind of come devices that have some "dynamic island-like" feauture, taht it gonna take too much space on the top. Anyway lets keep it, if you are confident it gonna work 👍.

Reply:
The background color does not behave as expected on my localhost :/. 

![Screenshot 2024-05-13 at 9 18 11](https://github.com/trezor/trezor-suite/assets/26143964/fff0ee0a-9005-4778-81bb-cf100d23b4c9)

Reply:
That is a Status bar. I'll check what can I do about that

Reply:
c784246629

Reply:
thanks. `statusBarTranslucent` is defined only for Android os. Did you test this on iOS?

Reply:
yes, you can check screenshots that are from iOS